mod/messageboard/actions/messageboard/delete.php

```diff
@@ -9,9 +9,7 @@
 $message = elgg_get_annotation_from_id($annotation_id);
 
 if ($message && $message->canEdit() && $message->delete()) {
-	system_message(elgg_echo("messageboard:deleted"));
-} else {
-	system_message(elgg_echo("messageboard:notdeleted"));
+	return elgg_ok_response('', elgg_echo('messageboard:deleted'));
```

Comment:
Works... is this what you had in mind?
![message_board_deleted](https://cloud.githubusercontent.com/assets/861833/20594656/5c99d5da-b237-11e6-937c-fd9fa6ffbe8c.png)